src/main/java/uk/nhs/digital/ps/migrator/model/hippo/NationalIndicatorMigrator.java

```diff
@@ -1,59 +1,86 @@
 package uk.nhs.digital.ps.migrator.model.hippo;
 
-import static java.util.stream.Collectors.toList;
-
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
 
-import org.apache.poi.ss.usermodel.Cell;
 import org.apache.poi.ss.usermodel.Row;
 
 import uk.nhs.digital.ps.migrator.config.ExecutionParameters;
 
 /**
  * This class is responsible for loading national indicators to import into hippo.
  */
-public class NationalIndicatorMigrator extends BaseXlsReader {
+public class NationalIndicatorMigrator {
 
     private ExecutionParameters executionParameters;
+    private XlsxReader xlsxReader;
+
     private static final String INDICATORS_SHEET_NAME = "Indicators";
-    private static final String TITLE_COLUMN = "Title";
 
-    public NationalIndicatorMigrator(ExecutionParameters executionParameters) {
+    private static final String IAPCODE_COLUMN = "IAPCode";
+    private static final String TITLE_COLUMN = "Title";
+    private static final String PUBLISHEDBY_COLUMN = "Published By";
+    private static final String PUBLISHEDDATE_COLUMN = "Published Date";    
+    private static final String REPORTINGPERIOD_COLUMN = "Reporting Period";   
+    private static final String REPORTINGLEVELS_COLUMN = "Reporting level(s)";  
+    private static final String BASEDONDATAFROM_COLUMN = "Based on data from";  
+    private static final String CONTACTAUTHOR_COLUMN = "Contact Author"; 
+    private static final String RATING_COLUMN = "Rating";     
+    private static final String ASSURANCEDATE_COLUMN = "Assurance date";   
+    private static final String REVIEWDATE_COLUMN = "Review date";       
+    private static final String INDICATORSET_COLUMN = "Indicator Set";  
+    private static final String PURPOSE_COLUMN = "Purpose";
+    private static final String DEFINITION_COLUMN = "Definition";
+    private static final String DESCRIPTOR_COLUMN = "Descriptor";    
+    private static final String INTERPRETATIONGUIDELINES_COLUMN = "Interpretation Guidelines";  
+    private static final String CAVEATS_COLUMN = "Caveats";  
+
+    private Path indicatorImportPath;
```

Comment:
This doesn't seem to need to be a field - as it's only being used in one method, it could be turned into a local variable.

Reply:
Sure. I think happened during moving logic around.